docs/charra-concept-on-pc.md

```diff
@@ -0,0 +1,28 @@
+# How to run CHARRA concept on PC
+
+The detailed steps of how to build and run CHARRA applications (Attester and Verifier) are
+discribed on corresponding GitHub repository:
+ - [Build and run CHARRA on PC](https://github.com/Fraunhofer-SIT/charra#build-and-run)
+
+In general there are two common ways to reach the goal:
+* Build docker container with binary packages
+* Build neccessary packages from sources inside docker container
+
+The first path is faster and more preferable. When the project is built one may run and test
+Attester and Verifier application:
+1. Configure network for docker in order to allow communicate containers.
+E.g. Edit file ./docker/run.sh and add internal virtual network:
+```
+--network=charra_default
```

Comment:
How to create this network?

Reply:
After I run `./docker/build.sh` script I have found this network... 
```shell
$ docker network ls
NETWORK ID     NAME             DRIVER    SCOPE
....
a13b7967a1d4   charra_default   bridge    local
....
```
And just specify its name in script `./docker/run.sh`

Reply:
Fixed